src/ocm/api/axiosClient.ts

```diff
@@ -41,6 +45,7 @@ export const setAuthInterceptor = (authInterceptor: (client: AxiosInstance) => A
     aiInterceptor(authInterceptor(axios.create())),
     axiosCaseConverterOptions,
   );
+  noCoverterClient = aiInterceptor(authInterceptor(axios.create()));
```

Comment:
`SetAuthInterceptor` is called by uhc-portal, the whole authentication handling is performed by them. Di you test this change didn't break anything in the uhc-portal?